internal/bridgeapi/client.go

```diff
@@ -361,15 +360,23 @@ func (c *Client) ClusterDetail(id string) (ClusterDetail, error) {
 		return ClusterDetail{}, errors.New("unexpected response status from API")
 	}
 
+	// transition type handles both pre-denested and nested unmarshal, leaving
+	// to remaining logic
 	var detail struct {
```

Comment:
Ah yeah I see. Okay as mentioned during meeting you don't need this — if you just replaced `var detail struct` with `var detail ClusterDetail` it'll work fine now or later, but it's harmful, so up to you.